templates/openstack/what-is-openstack.html

```diff
@@ -10,446 +13,556 @@
+        alt="Pacific Textiles Holdings Limited",
+        width="852",
+        height="97",
+        hi_def=True,
+        loading="lazy") | safe
+      }}
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>
+        In 2022, Pacific Textiles Holdings Limited &mdash; one of the world's largest fabric mills &mdash; announced that it's going to use Canonical’s <a href="/openstack">Charmed OpenStack</a> and <a href="/kubernetes">Charmed Kubernetes</a> to give its developers easy access to technologies like cloud-native apps, Kubernetes, AI/ML, and others alike.
+      </p>
+      <div class="p-cta-block">
+        <a href="/blog/pacific-textiles-drives-digital-transformation-with-new-infrastructure-for-legacy-and-cloud-native-applications" aria-label="Read the Pacific Textiles announcement">Read the announcement&nbsp;&rsaquo;</a>
+      </div>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_4' -%}
+      {{ image(url="https://assets.ubuntu.com/v1/96e9280c-logo-4.png",
+        alt="Nayatel",
+        width="852",
+        height="100",
+        hi_def=True,
+        loading="lazy") |safe  
+      }}
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_4' -%}
+      <p>
+        In 2022, Nayatel &mdash; one of Pakistan's leading telecom providers &mdash; announced that it completed the deployment of Pakistan’s first local public cloud. Aiming to meet the demand for a homegrown public cloud service, Nayatel partnered with Canonical to build an <a href="/openstack/features">enterprise-grade OpenStack platform</a>.
+      </p>
+      <div class="p-cta-block">
+        <a href="/engage/nayatel-casestudy-charmed-openstack">Read the case study&nbsp;&rsaquo;</a>
+      </div>
+    {%- endif -%}
+  {%- endcall -%}
+
+  <section class="p-section">
+    <hr class="p-rule is-fixed-width" />
+    <div class="p-section--shallow">
+      <div class="grid-row">
+        <div class="grid-col-4 grid-col-medium-2">
+          <h2>OpenStack is big</h2>
+        </div>
+        <div class="grid-col-4 grid-col-medium-2">
+          <p>
+            According to the 451 Research Market Monitor (Open Source Software, OpenStack) from September 2019, OpenStack's combined market size worldwide is $7.7B. It also continues to be one of the top three open-source projects in the world in terms of active contributions, along with the Linux kernel and Chromium. 
+          </p>
+          <p>
+            In 2020 OpenStack reached its 10-year anniversary which proves its maturity as a software project. Governed by the <a href="https://openinfra.dev/">Open Infrastructure Foundation</a>, more than 34,000 individual contributors and 550 companies participate in OpenStack development. It is also the most popular open-source private cloud platform and its adoption continues to grow.
+          </p>
+        </div>
       </div>
     </div>
+
     <div class="row u-hide--small u-hide--medium">
```

Comment:
GH won't let me leave the suggestion but assuming lines 564-578 can be deleted since we're not showing this chart